### PR TITLE
[step-sequencer] schedule sounds

### PR DESCRIPTION
**Pull request by sbungartz**
Hi,
first of all I want to thank you for putting together the fantastic article on [Advanced techniques](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Advanced_techniques) for the Web Audio API.
It helped me understand quite a lot about the Web Audio API.

Reading through the article I noticed that the scheduling introduced in the last paragraph of the example is not actually applied to the `play*()`-Functions: While the notes are enqueued and drawn with exact timing, the sounds themselves are still started at the time when they are scheduled.

Just as described in [A tale of two clocks](https://www.html5rocks.com/en/tutorials/audio/scheduling/), when you start resizing the browser window quickly you start to notice some glitching. To test it more easily I temporarily increased the `scheduleAheadTime` to five seconds or so.

This PR tries to fix that. With these changes I cannot notice glitches anymore.

I also removed the `TODO set value at time rather than value`, assuming it refers to exact change.

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
@Rumyra , can I ask you to review this one please?

**Comment by chrisdavidmills** [comment]
@Rumyra ping

**Comment by Rumyra** [comment]
I'm going to go over all web audio related issues first thing on Thursday 🙏

**Review by Rumyra — APPROVED** [review]
Yes this looks brilliant - thank you @sbungartz 🙏

**Comment by Rumyra** [comment]
@chrisdavidmills can I get write access to this repo? I just did a roundabout am I using this or my fork and moving forward it's probably best if I just work here 👍

Happy to merge after

**Comment by chrisdavidmills** [comment]
@Rumyra done! Let me know if you need anything else.

**Comment by Rumyra** [comment]
Awesome - thank you @chrisdavidmills !

PR here for article updates just FYI https://github.com/mdn/content/pull/2445

**Pull request merged**
